.yamllint.yaml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
*Commit message*
```
checks: dd yamllint config file 
```
"dd" should probably be "add"?

Reply:
fixed, thanks